### PR TITLE
Added id & class attribute + methods for data-attributes

### PR DESCRIPTION
**Pull request by jamowei**

### CODE REVIEW

**Comment by jamowei** [comment]
Added missing attributes

File: core/src/main/kotlin/io/fritz2/dom/tag.kt [hunk 1]
```diff
@@ -29,4 +30,11 @@ abstract class Tag<out T : Element>(tagName: String, override val domNode: T = w
     fun <X : Element> Flow<Tag<X>>.bind(): SingleMountPoint<WithDomNode<Element>> = DomMountPoint(this, domNode)
 
     fun <X : Element> Flow<Patch<Tag<X>>>.bind(): MultiMountPoint<WithDomNode<Element>> = DomMultiMountPoint(this, domNode)
+
+    operator fun <T> T.not() = Const(this)
```
Comment (jamowei) [inline core/src/main/kotlin/io/fritz2/dom/tag.kt]: I put it in here cause then no extra import is needed. So it's a bit easier to use ;)

**Comment by jwstegemann** [comment]
And now: an AttributeMountPoint for Seq (classes)

Jan <notifications@github.com> schrieb am Mo. 3. Feb. 2020 um 22:39:

**Comment by jamowei** [comment]
> And now: an AttributeMountPoint for Seq (classes) Jan <notifications@github.com> schrieb am 
Done

**Pull request merged**